resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -585,6 +593,7 @@ public class ApiClient implements AutoCloseable {
                     try {
                         ApiResponse<T> response;
                         try {
+                           //connectorResponse = retryOnFailure(connectorRequest, connectorResponse);
```

Comment:
Remove commented out code

Reply:
yes. Going to add Async version to that.